addon.xml

```diff
@@ -2,10 +2,18 @@
 <addon id="script.module.inputstreamhelper" version="0.3.5" name="InputStream Helper" provider-name="emilsvennesson">
     <requires>
         <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.inputstreamhelper" version="0.3.5" optional="true"/>
```

Comment:
Why is this needed? :frowning:

Reply:
Otherwise our "plugin" or "script" would not be able to import inputstreamhelper.
It looks silly, but it works.